library/build.gradle.kts

```diff
@@ -50,33 +52,44 @@ android {
             suppressWarnings = false
         }
     }
+
+    // Custom ktlint script
+    tasks.register("ktlint") {
+        logger.lifecycle("ktlint")
+        exec {
+            commandLine = listOf("./../scripts/lint.sh", "--no-error")
+        }
+    }
+
+    // Make ktlint run before build
+    tasks.getByName("preBuild").dependsOn("ktlint")
 }
 
 dependencies {
     // Bundled
-    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Constants.BuildScript.kotlinVersion}")
-    implementation("androidx.annotation:annotation:1.5.0")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Constants.BuildScript.kotlinVersion}")
```

Comment:
Change this to `kotlin-stdlib` (since Kotlin 1.8.0 all artifacts are merged).
https://kotlinlang.org/docs/whatsnew18.html#updated-jvm-compilation-target

Reply:
👍🏻